medusa/show_queue.py

```diff
@@ -437,6 +438,19 @@ def run(self):
                 return
 
         # TODO: Add more specific indexer exceptions, that should provide the user with some accurate feedback.
+        except IndexerShowNotFound as e:
```

Comment:
A while ago we decided to not use the single char variables anymore. Als not for obvious ones like exceptions. Can you chance them to `error`